crypto/crypto.go

```diff
@@ -26,11 +26,19 @@ type PubKey interface {
 	Equals(PubKey) bool
 }
 
+// this interface will be phased out, this is here for backwards compatibility where needed.
+// Bytes in tendermint 0.33 and before uses marshalbinarybare from amino, this function should provide the
+// bytes as return and not marshal.
+type AminoBytes interface {
+	AminoBytes() []byte
+}
+
```

Comment:
@alexanderbez you won't need this if I just return []byte of the key and not marshal anything, correct?

Reply:
We'd just use the raw type `[]byte`. We wouldn't call any methods. But this might be moot as we're discussing move key types to the SDK.